apis/core/bootstrap/lib/query.ts

```diff
@@ -3,5 +3,6 @@ import $rdf from 'rdf-ext'
 import { SparqlTemplateResult } from '@tpluscode/rdf-string'
 
 export function lindasQuery(query: SparqlTemplateResult): NamedNode {
-  return $rdf.namedNode('https://register.ld.admin.ch/query?query=' + encodeURIComponent(query.toString()))
+  // TODO: Switch to production system once stable.
+  return $rdf.namedNode('https://test.lindas.admin.ch/query?query=' + encodeURIComponent(query.toString()))
```

Comment:
What's this about?

Reply:
The current production endpoint gave me randomly 3 different results on the same query. I expect it to be solved with the migration to V7. So I don't want to spend more time here.

Reply:
Note that the `lindasQuery` function is also used in other places (themes?). I hope the test endpoint also contains this data.

Reply:
yes, thank you, currently all "metadata" is in sync on all environments. so its save as long we use it not for other data. shall we add a comment in the code?